pgmax/fg/groups.py

```diff
@@ -538,6 +287,10 @@ def __post_init__(self):
             if not hasattr(self, key):
                 object.__setattr__(self, key, getattr(self.factor, key))
 
+        object.__setattr__(
+            self, "log_potentials", getattr(self.factor, "log_potentials")
+        )
```

Comment:
Why didn't we have this before?

Reply:
because for SingleFactorGroup, we were calling `factor.compile_wiring`

Reply:
Still not sure this is needed...